safe-global/safe-wallet-web
# fix: Blockaid warning styles

## What it solves

Resolves #4190

## How this PR fixes it

- Uses `static.main` for the text color
- Uses `currentColor` for the blockaid logo
- Overwrites the checkbox color to `static.main`
- font-weight bold for the warning title
- Reduces spacing of the list items in the warning

## How to test it

1. Create a transaction on mainnet to transfer funds to an empty EOA or create any other transaction that triggers a warning
2. Observe the style looks readable in both light and dark mode

## Screenshots
<img width="658" alt="Screenshot 2024-11-06 at 20 14 13" src="https://github.com/user-attachments/assets/bfe54d7d-a4ef-4a06-8d4a-9e43bbc16dd1">
<img width="652" alt="Screenshot 2024-11-06 at 20 14 08" src="https://github.com/user-attachments/assets/4df56493-ed60-41c1-afd0-0c55fd0fcdf1">

## Checklist
